modules/mono/glue/GodotSharp/GodotSharp/Core/Bridge/GCHandleBridge.cs

```diff
@@ -18,5 +18,30 @@ internal static void FreeGCHandle(IntPtr gcHandlePtr)
                 ExceptionUtils.LogException(e);
             }
         }
+
+        [UnmanagedCallersOnly]
+        internal static godot_bool GCHandleIsTargetCollectible(IntPtr gcHandlePtr)
+        {
+            try
+            {
+                var target = GCHandle.FromIntPtr(gcHandlePtr).Target;
+
+                if (target is null)
+                    return godot_bool.True;
+
+                if (target.GetType().Assembly.IsCollectible)
+                    return godot_bool.True;
+
+                if (target is Delegate @delegate && DelegateUtils.IsDelegateCollectible(@delegate))
+                    return godot_bool.True;
```

Comment:
Can the target be collectible if the type is not in a collectible assembly?

Reply:
Yes, for example if the delegate is of type `Action`, then the type will not be reloadable as it is in the corelib, but the target and method can be in a reloadable assembly.